### ISSUE TITLE
Publishing: revert to using .props file with version of Tasks.Feed package

### ISSUE THREAD

**Issue by JohnTortugo** [dotnet/arcade#1982]
Currently the release pipeline import the DefaultVersions.props and Version.props files; this is causing an issue as DefaultVersion.props imports NugetFile.config. 

Also, after recent changes on Arcade-validation mechanism we can now go back to the preferable option of just putting MicrosoftDotNetTasksFeedVersion on a .props file. This issue is to revert the release pipeline to just import the Tasks.Feed version from a .props file.

Once this is done we can remove the ` /p:RestoreConfigFile=` from the release pipeline configuration.

cc/ @riarenas

**Comment by riarenas** [comment]
Also taking this over from Cesar. The workaround we're using is good enough for now, so I'll tackle this once I'm done with #1994

**Comment by tmat** [comment]
@JohnTortugo 

This issue is linked from DefaultVersions.props:

https://github.com/dotnet/arcade/blob/master/src/Microsoft.DotNet.Arcade.Sdk/tools/DefaultVersions.props#L15-L17

The issue is closed by the conditions on the imports were not removed. Can these conditions be removed now?

**Comment by JohnTortugo** [comment]
We still rely on DefaultVersions.props, so we still need those conditions. I created a new issue to track this refactoring: https://github.com/dotnet/arcade/issues/2809
